docs/pages/access-controls/getting-started.mdx

```diff
@@ -21,12 +20,12 @@ with creating your own role.
 
 ## Step 1/3. Add local users with preset roles
 
-Teleport provides [several preset roles](./reference.mdx#preset-roles):
-`editor`, `auditor` and `access`. Members of the `editor` role can modify
-cluster configuration, members of `auditor` role can view audit logs, and
-`access` members can access cluster resources.
+Teleport provides several preset roles: `editor`, `auditor` and `access`.
+Members of the `editor` role can modify cluster configuration, members of
```

Comment:
```suggestion
Users with the `editor` role can modify cluster configuration, users with
```

Similar to above - I think it's best to describe roles as something assigned to users, instead of "groups" that you join or become a member of.

Reply:
To further clarify roles vs. groups, just for my own understanding as well:

Roles are sets of rights. Groups are sets of people.

Roles can also be functions that one person or groups of people can be permitted to do.